.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Deploy
+on: [push]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+        with:
+          persist-credentials: false
+
+      - name: Setup node 🎱
+        uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+
+      - name: Install 🎢
+        run: yarn install --force
+
+      - name: Build 🧩
+        run: yarn run build:static
```

Comment:
In error mide dige.
`error Command "build:static" not found.` mige dige, in command to nadari,
bayad toye bakhshe: `scripts: ` package.json add `build:static` midashti.
ya age project react hast, ino avaz kon `yarn build` koni